extensions/tmdb/src/components/Backdrops.tsx

```diff
@@ -0,0 +1,102 @@
+import { ActionPanel, Action, Grid, Icon, showToast, Toast } from "@raycast/api";
+import { moviedb } from "../api";
+import { MovieImagesResponse, TvImagesResponse } from "moviedb-promise";
+import { useCachedPromise } from "@raycast/utils";
+import { useState } from "react";
+
+function Backdrops({ id, type }: { id: number; type: "movie" | "tv" }) {
+  const [selectedLanguage, setSelectedLanguage] = useState<string>("all");
+
+  const fetchBackdropsAndLanguages = async () => {
```

Comment:
So when you look at the documentation of [useCachedPromise](https://developers.raycast.com/utilities/react-hooks/usecachedpromise), you'll see that the function that is eventually called, should have the parameters that are used.

In this case it would be `const fetchBackdropsAndLanguages = async(id: number; type: "movie" | "tv") => {`